static/src/javascripts/projects/common/modules/commercial/adblock-messages.js

```diff
@@ -9,18 +9,18 @@ define([
     storage,
     userFeatures
 ) {
-    function sharedRules() {
-        var alreadyVisted = storage.local.get('gu.alreadyVisited') || 0;
-
-        return detect.getBreakpoint() !== 'mobile' && detect.adblockInUse() && config.switches.adblock && alreadyVisted > 1;
-    }
-
     function noAdblockMsg() {
```

Comment:
I realize that you're calling the localStorage value multiple times so that you can make this module testable. But... I think the way we declare and use modules is a bit awkward, really. All of our services are written as stateful singletons, usually with init methods that we trigger in our tests and only-by-convention trigger once in our production code. It would seem much cleaner if we just had constructors for our services that could be instantiated in a service locator...